test/run_pass/test_net_http_server.js

```diff
@@ -181,7 +183,7 @@ finalReq.end();
 var server2 = http.createServer();
 
 // Create server instance without new keyword.
-var server3 = Server(function(request, response) {});
+// var server3 = Server(function(request, response) {});
```

Comment:
Why this is commented out?

Reply:
I forgot to uncomment it, but it works. I'll fix it.